steward_proto/src/prost/cellars.uniswapv3.v1.rs

```diff
@@ -18,6 +18,17 @@ pub struct RebalanceRequest {
 }
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct RebalanceResponse {}
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
We need to re-run the proto build now that we have deleted the new proto, so that this file isn't including functions for a proto that doesn't exist any more (and that will also fix `descriptor.bin`, which should not be changing in this PR.